packages/node-opentelemetry/src/otel.ts

```diff
@@ -99,6 +101,9 @@ export const initSDK = (config: SDKConfig) => {
         },
         ...config.instrumentations,
       }),
+      ...(config.additionalInstrumentations
+        ? config.additionalInstrumentations
+        : []),
```

Comment:
what about
```
...(config.additionalInstrumentations ?? [])
```

Reply:
Oh yeah that makes more sense, been a while since Ive Typescripted so Ill default to your knowladge on this one haha